oio/common/storage_method.py

```diff
@@ -15,6 +15,7 @@
 
 import sys
 from oio.common import exceptions
+from pyeclib.ec_iface import ECDriver, ECDriverError
```

Comment:
IIRC this import was loaded only when needed to avoid slow loading and/or logs regarding missing libs

Reply:
Indeed, it would we safer to load it lazily, or the whole module won't be loadable when pyeclib or liberasurecode are not installed.

Reply:
When we import a module in `spawn_n`, sometimes the import doesn't work.
```
[...]
Checking object "account=myaccount, container=toto30, obj=30"
Checking object "account=myaccount, container=toto74, obj=74"
Traceback (most recent call last):
  File "oio-venv/local/lib/python2.7/site-packages/eventlet/greenpool.py", line 82, in _spawn_n_impl
    func(*args, **kwargs)
  File "oio-sds/oio/crawler/integrity.py", line 298, in check_obj
    self._check_obj_policy(target, meta, results, recurse=recurse)
  File "oio-sds/oio/crawler/integrity.py", line 241, in _check_obj_policy
    stg_met = STORAGE_METHODS.load(obj_meta['chunk_method'])
  File "oio-sds/oio/common/storage_method.py", line 65, in load
    self.cache[chunk_method] = cls.build(params)
  File "oio-sds/oio/common/storage_method.py", line 175, in build
    ec_nb_parity=ec_nb_parity)
  File "oio-sds/oio/common/storage_method.py", line 157, in __init__
    from pyeclib.ec_iface import ECDriver, ECDriverError
ImportError: cannot import name ECDriver
Checking container "account=myaccount, container=toto86"
Checking container "account=myaccount, container=toto87"
[...]
```

Reply:
Then, we will have to find a workaround.

Reply:
With my workaround:
```
> openio object create fve Makefile 
Failed to upload Makefile in fve: 'liberasurecode_rs_vand' (ECDriverError: Erasure coding not available: liberasurecode.so.1: cannot open shared object file: No such file or directory) Check erasure code packages.
+----------+------+------+--------+
| Name     | Size | Hash | Status |
+----------+------+------+--------+
| Makefile |    0 | None | Failed |
+----------+------+------+--------+
Too many errors occured
> openio object create fve Makefile --policy THREECOPIES
+----------+-------+----------------------------------+--------+
| Name     |  Size | Hash                             | Status |
+----------+-------+----------------------------------+--------+
| Makefile | 43319 | 7728225C739685F2E3AE8F75418CFE38 | Ok     |
+----------+-------+----------------------------------+--------+
```